android/src/main/kotlin/com/weplenish/flutter_wifi_connect/FlutterWifiConnectPlugin.kt

```diff
@@ -355,8 +375,10 @@ class FlutterWifiConnectPlugin() : FlutterPlugin, MethodCallHandler {
     val intentFilter = IntentFilter()
     intentFilter.addAction(WifiManager.NETWORK_STATE_CHANGED_ACTION)
     context?.registerReceiver(wifiChangeReceiver, intentFilter)
-
-    wifiManager.disconnect()
+    // remove network to emulate a behavior as close as possible to new Android API
+    wifiManager.removeNetwork(network)
```

Comment:
wondering if we want to honour the `joinOnce` argument here... The implemented behaviour would be that of `joinOnce: true`

Reply:
We probably should so it aligns with iOS. But this is fine for now.